### PR TITLE
Adjust styling

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ export default createGlobalStyle`
   box-sizing: border-box;
 };
 
+
 body {
   font-family:  -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  background: rgba(235, 240, 242, 0.2);
+  color: rgba(71, 39, 35, 0.72);
   font-size: 18px;
```
Comment (MichiBerger) [inline src/GlobalStyles.js:14]: e.g. you could give it a font-size: 112.5%;
but it´s the same 👍  so doesn´t matter ;)

File: src/components/ActivityCard.js [hunk 2]
```diff
@@ -20,16 +22,9 @@ export default function Card({
...
-        <Button
-          background="transparent"
-          justifySelf="end"
-          padding="10px"
-          width="auto"
-          height="auto"
-          onClick={() => setShow(true)}
-        >
+        <DeleteButton onClick={() => setShow(true)}>
           <img src={deleteicon} alt="delete" />
```
Comment (MichiBerger) [inline src/components/ActivityCard.js:26]: depends how you want to style your icons...you could go inline svg here or even create a icon component! advantage is that you have more control of styling...

File: src/components/ActivityCard.js [hunk 3]
```diff
@@ -44,7 +39,6 @@ export default function Card({
 }
 
 const WrapperCard = styled.section`
-  color: rgba(71, 39, 35, 0.72);
   border-bottom: 0.5px solid rgba(71, 39, 35, 0.72);
```
Comment (MichiBerger) [inline src/components/ActivityCard.js:42]: pretty thin border...maybe at least 1px! I´ve just remember that gregor said somthing that 0.5px should be avoided!? but not sure

File: src/components/ActivityCard.js [hunk 4]
```diff
@@ -53,7 +47,7 @@ const WrapperCard = styled.section`
 `;
 
 const LinkStyling = styled(Link)`
-  padding: 8px 8px 0;
+  padding: 8px 8px 5px;
   font-size: 18px;
```
Comment (MichiBerger) [inline src/components/ActivityCard.js:51]: you could move your font-size to global styles for all links...just check I think it´s an a tag but not 100% sure...but it´s not necessary ;)

File: src/components/Button.js [hunk 5]
```diff
@@ -1,46 +1,60 @@
...
+`;
+export { EditButton };
+
+const DeleteButton = styled(Button)`
+  background: transparent;
+  justify-self: end;
+  align-self: flex-end;
+  width: auto;
+  height: auto;
+`;
```
Comment (MichiBerger) [inline src/components/Button.js:59]: I like the idea to have a own button component and there all different buttons! nice ;)

File: src/components/Button.stories.js [hunk 6]
```diff
@@ -1,8 +1,10 @@
 import Button from './Button.js';
+import { IconButton } from './Button.js';
+
 import deleteicon from '../images/binicon.svg';
+import editicon from '../images/edit.svg';
 import gobackicon from '../images/goback.svg';
 import newicon from '../images/new.svg';
```
Comment (MichiBerger) [inline src/components/Button.stories.js:7]: for readability I would recommend camelCase e.g. goBackIcon or deleteIcon

File: src/components/DeleteModal.js [hunk 7]
```diff
@@ -1,60 +1,47 @@
-import Button from './Button.js';
 import styled from 'styled-components';
 
+import { ModalButton } from './Button.js';
+
 export default function DeleteModal(props) {
   if (!props.show) {
```
Comment (MichiBerger) [inline src/components/DeleteModal.js:6]: here you could destructure you show in `DeleteModal({show})` and then `if (!show) {`

File: src/components/DeleteModal.js [hunk 8]
```diff
@@ -1,60 +1,47 @@
...
-      <div>
-        <Button
-          width="fit-content"
-          height="fit-content"
-          fontSize="14px"
-          type="submit"
-          onClick={props.onClose}
-        >
+      <span>
+        <ModalButton type="submit" onClick={props.onClose}>
```
Comment (MichiBerger) [inline src/components/DeleteModal.js:13]: same as on the top ;) so you don´t have to write always `props.`

File: src/components/DeleteModal.js [hunk 9]
```diff
@@ -1,60 +1,47 @@
...
-        <Button
-          width="fit-content"
-          height="fit-content"
-          fontSize="14px"
-          type="submit"
-          margin="15px"
-          onClick={props.onDelete}
-        >
+        </ModalButton>
+        <ModalButton type="submit" onClick={props.onDelete}>
```
Comment (MichiBerger) [inline src/components/DeleteModal.js:16]: same as on the top ;) so you don´t have to write always props.

File: src/components/Picture.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+import backgroundpicture from '../images/background.svg';
```
Comment (MichiBerger) [inline src/components/Picture.js:3]: I would recommend camelCase ;)

File: src/GlobalStyles.js (hunk 1 above)
Comment (JelenadlS) [reply src/GlobalStyles.js:14]: Well, didnt thought about that, but i think i would keep it as it is now - completely used to px🙈 Anyway, thank you!😊

File: src/components/ActivityCard.js (hunk 2 above)
Comment (JelenadlS) [reply src/components/ActivityCard.js:26]: Thats a good idea, I also checked how you did it 😊
I think i will keep it for now like this, as i am using them straight away as they are and actually not really style them..

File: src/components/ActivityCard.js (hunk 3 above)
Comment (JelenadlS) [reply src/components/ActivityCard.js:42]: damn, completely oversaw this one 😅
Thought I adjusted all of Gregors comments - THANK YOU!

File: src/components/ActivityCard.js (hunk 4 above)
Comment (JelenadlS) [reply src/components/ActivityCard.js:51]: Uhh thank you, i thought i took it out everywhere as i actually put it into global styles😊

File: src/components/DeleteModal.js (hunk 7 above)
Comment (JelenadlS) [reply src/components/DeleteModal.js:6]: well you are right, especially as i already handed it over haha🤩

File: src/components/Button.stories.js [hunk 11]
```diff
@@ -1,67 +1,58 @@
...
+
+export const ModalKeep = args => (
+  <ModalButton {...args}>no, I want to keep it</ModalButton>
+);
+
+export const Arrowback = args => (
+  <ArrowbackButton {...args}>
+    <img src={goBackIcon} alt="goback" />
+  </ArrowbackButton>
+);
```
Comment (ttaannjjaa) [inline src/components/Button.stories.js:46]: This comment i only make as i saw quite at the end of the review files that you wrote goBackIcon - so maybe you like to change also ArrowbackButton to ArrowBackButton - i think both versions can be used

**Review by ttaannjjaa — APPROVED** [review]
Great work - looks very good to me

**Pull request merged**